### ISSUE TITLE
Dev Container integration

### ISSUE THREAD

**Issue by mathiasmaerker** [microsoft/devhome#1211]
### Suggested new feature or improvement

The overall trend is to provide remote and or local developer experience via Dev Containers. Currently (if I understand correctly) the benefits of a Dev Drive would not apply to a Dev Container, because it uses the internal file system of the container. It would be nice to have a tight integration between the two technologies

### Scenario

see above

**Comment by thawkins** [comment]
Agreed, this would be awesome if it could optionally pick up a .devcontainer folder in a repo and apply the devcontainer.json file to load all the required tooling.

Alternatively allow the automatic creation of a dev container in wsl2, with designated tooling applied
